clusterloader2/testing/load/config.yaml

```diff
@@ -20,7 +20,7 @@
 {{$ENABLE_CONFIGMAPS := DefaultParam .ENABLE_CONFIGMAPS false}}
 {{$ENABLE_DAEMONSETS := DefaultParam .ENABLE_DAEMONSETS false}}
 {{$ENABLE_JOBS := DefaultParam .ENABLE_JOBS false}}
-{{$ENABLE_PVS := DefaultParam .ENABLE_PVS false}}
+{{$ENABLE_PVS := DefaultParam .CL2_ENABLE_PVS true}}
```

Comment:
Let's just change the name, don't change the default yet. If you change the default it will break the kubemark tests. 
Adding
/ok-to-test
to let you see that the kubemark presubmit will fail (unless you change the default back to false)

Also, would it be a problem for you to update all variables (add prefix) for consistency? It'd be really great if you could do it

Reply:
Thanks, I will add the prefix to all variables defined under https://github.com/kubernetes/perf-tests/tree/master/clusterloader2/testing/load/experimental/overrides

Reply:
SG, thanks!

Reply:
Done. Request you to please review if anything needs to be modified. Thanks Mani.